ansible/roles/oracle-db-monitoring/tasks/install-oracle-db-monitoring.yml

```diff
@@ -1,7 +1,11 @@
-- name: Get monitoring tags as a list
+- name: Get oracle database sids for monitoring as a list from the oracle-sids tag
   set_fact:
     oracle_monitoring_list: "{{ ec2.tags['oracle-sids'].split() | default([]) }}"
 
+- name: Get the monitoring target for checking the hosts connection to fixngo
```

Comment:
Will this work if that tag is not present?  I have a feeling it will bail out with an error

Reply:
added a conditional here